discovery-server/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.4.2</version>
```

Comment:
This isn't the version we use on the other modules. Can you refer to the config server and check both the spring-boot version and the cloud version?

Reply:
Yes you are right, I though I fixed it but apparently it was in the second PR. Changed to version 2.3.6.RELEASE. The cloud version is okay, Hoxton.SR9